cmd/plume/release.go

```diff
@@ -55,6 +62,14 @@ func init() {
 }
 
 func runRelease(cmd *cobra.Command, args []string) {
+	if systemInfo, ok := releaseSystems[selectedSystem]; !ok {
+		plog.Fatal("Unknown system %q", selectedSystem)
+	} else if err := systemInfo.handler(cmd, args); err != nil {
+		plog.Fatal(err)
+	}
+}
+
+func runCLRelease(cmd *cobra.Command, args []string) error {
```

Comment:
There are a lot of remaining `plog.Fatal` calls still inside this function that can be changed to just returning the error and letting https://github.com/coreos/mantle/pull/934/commits/f73d8de4ade96285f13eb2864fe6e29a1e118cea#diff-70d5486f73eaac934075d6c18abdcc27R68 take care of it.